androidtv/constants.py

```diff
@@ -28,20 +28,32 @@
 #: Determine whether the device is awake
 CMD_AWAKE = "dumpsys power | grep mWakefulness | grep -q Awake"
 
-#: Get the current app
-CMD_CURRENT_APP = "CURRENT_APP=$(dumpsys window windows | grep mCurrentFocus) && CURRENT_APP=${CURRENT_APP#*{* * } && CURRENT_APP=${CURRENT_APP%%/*} && echo $CURRENT_APP"
+#: Parse the current app from the dumpsys output and store it in the variable ``CURRENT_APP``
```

Comment:
Minor suggestion, to be more in accordance with `CMD_LAUNCH_APP_CONDITION` and `CMD_MEDIA_SESSION_STATE` comments, stating which variables are assumed to be defined/set prior executing command.

```suggestion
#: Parse current application identifier from dumpsys output and assign it to ``CURRENT_APP`` variable (assumes dumpsys output is momentarily set to ``CURRENT_APP`` variable)
```